### PR TITLE
add hometask-1

### PR DESCRIPTION
**Pull request by Ikl0**

### CODE REVIEW

**Review by MariaGrechko — APPROVED** [review]

File: ikl0/hometask-1/task2.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+arr = [1, 2]
+result = 2
+while arr[1] <= 4000000 do
```
Comment (fooooxmr) [inline ikl0/hometask-1/task2.rb:4]: 4_000_000

File: ikl0/hometask-1/task4.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
+max = -1
+999.downto(100) do |i|
+  999.downto(100) do |x|
+    num = i*x
+    if num.to_s == num.to_s.reverse
+      pal=num
+      if pal.to_i > max
```
Comment (fooooxmr) [inline ikl0/hometask-1/task4.rb:7]: One line if.

File: ikl0/hometask-1/task4.rb [hunk 3]
```diff
@@ -0,0 +1,13 @@
+max = -1
+999.downto(100) do |i|
+  999.downto(100) do |x|
+    num = i*x
+    if num.to_s == num.to_s.reverse
+      pal=num
```
Comment (fooooxmr) [inline ikl0/hometask-1/task4.rb:6]: spaces

File: ikl0/hometask-1/task5.rb [hunk 4]
```diff
@@ -0,0 +1,2 @@
+#require 'rational'
```
Comment (fooooxmr) [inline ikl0/hometask-1/task5.rb:1]: Remove comment.

File: ikl0/hometask-1/task5.rb [hunk 5]
```diff
@@ -0,0 +1,2 @@
...
+p (1..20).inject(1){ |result,n| result.lcm n }
```
Comment (fooooxmr) [inline ikl0/hometask-1/task5.rb:2]: Understand inject and lcm command.

**Review by fooooxmr — APPROVED** [review]

**Pull request merged**